src/components/commons/banner/WarningBanner.tsx

```diff
@@ -8,24 +8,59 @@ interface WarningBannerProps {
   mainnet?: boolean
 }
 
+interface Announcement {
+  content: string
+  url?: string
+}
+
 export function WarningBanner (props: PropsWithChildren<WarningBannerProps>): JSX.Element {
-  const network = useNetwork().name
+  const deFiChainStatusUrl = 'https://status.defichain.com/'
 
-  if (props.mainnet === undefined && props.testnet === undefined) {
-    return <></>
+  const blockchainIsDownContent: Announcement = {
+    content: 'We are currently investigating a syncing issue on the blockchain. ',
+    url: deFiChainStatusUrl
   }
-
-  if (network !== 'mainnet' && props.mainnet!) {
-    return <></>
+  const oceanIsDownContent: Announcement = {
+    content: 'We are currently investigating connection issues on Ocean API. ',
+    url: deFiChainStatusUrl
   }
 
-  if (network !== 'testnet' && props.testnet!) {
+  const { isBlockchainDown, isOceanDown } = useApiStatus()
+  const [displayAnnouncement, setDisplayAnnouncement] = useState<Announcement | undefined>()
+
+  useEffect(() => {
+    if (isBlockchainDown) {
+      setDisplayAnnouncement(blockchainIsDownContent)
+    } else if (isOceanDown) {
+      setDisplayAnnouncement(oceanIsDownContent)
+    } else {
+      setDisplayAnnouncement(undefined)
+    }
+  }, [isBlockchainDown, isOceanDown])
+
+  // don't display banner when there is not announcement to display
+  if (displayAnnouncement === undefined) {
     return <></>
   }
 
   return (
-    <div className='bg-orange-100 rounded p-3 text-center' data-testid={props.testId}>
-      {props.children}
+    <div className='bg-orange-100 rounded p-3 text-center flex-row text-center' data-testid='warning_banner'>
+      <div>
+        {displayAnnouncement.content}
```

Comment:
Also, you can put this outside, so you don't need to add space on label. Just add padding/margin